packages/components/pie-button/package.json

```diff
@@ -5,9 +5,8 @@
   "type": "module",
   "main": "dist/index.js",
   "module": "dist/index.js",
-  "types": "dist/types/index.d.ts",
   "scripts": {
-    "build": "run -T vite build",
+    "build": "yarn build:wrapper pie-button && run -T vite build",
```

Comment:
not temporary but could be revisited - I added the `pie-button` as a parameter so only the pie-button wrapper gets built when inside that directory or when built from the root. Otherwise, yeah it does become generic and generates the wrappers for all components - maybe in future we could just do this but it might be confusing running yarn build in `pie-button` and having a wrapper created in `pie-modal` too 😄